api/v0/ingest/pb/ingest.proto

```diff
@@ -20,6 +20,8 @@ message IngestMessage {
         REGISTER_PROVIDER_RESPONSE=8;
         REMOVE_PROVIDER = 9;
         REMOVE_PROVIDER_RESPONSE=10;
+        INDEX_CONTENT=11;
+        INDEX_CONTENT_RESPONSE=12;
```

Comment:
Just an FYI: in the reference_provider we chose to create an `UNSET = 0` type so by default the message type is `UNSET`. Just so it doesn't default to `ERROR_MESSAGE`. Its a nit-pick and I don't think we need to change it here, but so you know.

Reply:
Do we have or need to have an error message for the reference provider?  If so, maybe change that to 0?